src/handlers/RPInitiatedLogoutRequest.js

```diff
@@ -40,20 +41,17 @@ class RPInitiatedLogoutRequest extends BaseRequest {
    * @returns {Promise}
    */
   static handle (req, res, provider) {
-    const { host } = provider
     const request = new RPInitiatedLogoutRequest(req, res, provider)
 
     return Promise
       .resolve(request)
       .then(request.validate)
-
       // From: https://openid.net/specs/openid-connect-session-1_0.html#RPLogout
       // At the logout endpoint, the OP SHOULD ask the End-User whether they want
       // to log out of the OP as well. If the End-User says "yes", then the OP
       // MUST log out the End-User.
-      .then(host.logout)
```

Comment:
Why was the `host.logout` callback removed? That's a key part of the mechanism.

Reply:
Ok, got it. Let me look around the repos some more, maybe see if there's a clearer way..